pkg/workload/tpcc/tpcc.go

```diff
@@ -809,6 +826,18 @@ func (w *tpcc) Ops(
 		}
 	}
 
+	if duration, err := w.flags.GetDuration("duration"); err == nil {
+		if duration == 0 || duration >= longDurationWorkloadThreshold {
+			log.Warningf(ctx,
+				"Since the workload is set for %s duration and it's not meant for a public benchmark, "+
```

Comment:
How about "Warning: this workload is being used with a long or indefinite duration. This could cause it to deviate from the TPCC spec in subtle ways and is not meant for benchmarking. Consistency checks might be disabled"